src/scan-imports.ts

```diff
@@ -37,7 +37,7 @@ export type InstallTarget = {
 };
 
 function stripJsExtension(dep: string): string {
-  return dep.replace(/\.m?js$/i, '');
+  return dep.replace(/\.m?js$/i, ''); // otherwise, strip the extension
```

Comment:
not sure I follow this comment

Reply:
Whoops! That was leftover from a previous rev where this function was changed. Will remove; you’re right it doesn’t any make sense now.